conans/client/tools/apple.py

```diff
@@ -39,6 +35,17 @@ def apple_sdk_name(settings):
                 'tvOS': 'appletvos'}.get(str(os_), None)
 
 
+def apple_sdk_name(settings):
+    """returns proper SDK name suitable for OS and architecture
+    we're building for (considering simulators)"""
+    arch = settings.get_safe('arch')
+    os_ = settings.get_safe('os')
+    os_sdk = settings.get_safe('os.sdk')
+    if os_sdk:
+        return os_sdk.lower()
```

Comment:
If in every single place the sdk is used is lowercase, wouldn't it be better to define them as lowercase directly in the settings?

Reply:
I'd personally prefer always lower-case as it's less error-prone. @theodelrieu  WDYT?

Reply:
@SSE4 Agreed, lowercase looks good.

Reply:
changed to lowercase